Source/Core/AudioCommon/CubebStream.cpp

```diff
@@ -67,6 +67,11 @@ bool CubebStream::Init()
 
 bool CubebStream::SetRunning(bool running)
 {
+  if (m_running == running)
+    return false;
+
+  m_running = running;
```

Comment:
m_running should only be set after checking the returnvalue of `cubeb_stream_{start,stop}`. Otherwise the stream will still be considered even if for whatever reason it returns an error and the stream doesn't start.